### ISSUE TITLE
PHP issues

### ISSUE THREAD

**Issue by HardeepAsrani** [Codeinwp/teammates#13]
Plugin generates:

```
#teammates-section .teammates-members-wrapper a button {
    background: <br />
<b>Notice</b>:  Undefined variable: buttons_color in <b>/var/www/html/wordpress/wp-content/plugins/teammates/admin/inc/custom-colors.php</b> on line <b>18</b><br />
 !important;
  color: <br />
<b>Notice</b>:  Undefined variable: buttons_text_color in <b>/var/www/html/wordpress/wp-content/plugins/teammates/admin/inc/custom-colors.php</b> on line <b>19</b><br />
 !important;
}

#teammates-section .teammates-members-wrapper a button:hover {
    background: <br />
<b>Notice</b>:  Undefined variable: buttons_hover_color in <b>/var/www/html/wordpress/wp-content/plugins/teammates/admin/inc/custom-colors.php</b> on line <b>23</b><br />
 !important;
  color: <br />
<b>Notice</b>:  Undefined variable: buttons_hover_text_color in <b>/var/www/html/wordpress/wp-content/plugins/teammates/admin/inc/custom-colors.php</b> on line <b>24</b><br />
 !important;
}

#teammates-section .teammates-members-wrapper a {
    color: <br />
<b>Notice</b>:  Undefined variable: links_color in <b>/var/www/html/wordpress/wp-content/plugins/teammates/admin/inc/custom-colors.php</b> on line <b>28</b><br />
;
}
```

**Comment by abaicus** [comment]
Yes, I removed those options from the customizer but forgot to remove them from the php file. Thanks!

**Comment by abaicus** [comment]
Fixed in facd522ae5eb52b8a82aeaa3970d5d5a734e3acf.
